$NetBSD: patch-Makefile,v 1.2 2018/02/06 10:14:29 jperkin Exp $

* Let pkgsrc pull in the correct curses library
* Honor pkgsrc man directory

--- Makefile.orig	2016-11-03 07:34:00.000000000 +0000
+++ Makefile
@@ -44,7 +44,7 @@ ifeq ($(DEBUG),1)
 	LDFLAGS :=
 endif
 
-LDLIBS   = -l$(CURSES)
+LDLIBS   = -lcurses
 
 all: $(TARGET)
 
@@ -61,8 +61,8 @@ clean:
 install:
 	install -d $(DESTDIR)$(PREFIX)/bin
 	install -m 755 $(TARGET) $(DESTDIR)$(PREFIX)/bin/$(TARGET)
-	install -d $(DESTDIR)$(PREFIX)/share/man/man1
-	install -m 644 mdp.1 $(DESTDIR)$(PREFIX)/share/man/man1/$(TARGET).1
+	install -d $(DESTDIR)$(PREFIX)/${PKGMANDIR}/man1
+	install -m 644 mdp.1 $(DESTDIR)$(PREFIX)/${PKGMANDIR}/man1/$(TARGET).1
 
 uninstall:
 	$(RM) $(DESTDIR)$(PREFIX)/$(TARGET)
